Src/library/memory_analyzer/singletons/ma_shared_main_window.e

```diff
@@ -0,0 +1,68 @@
+note
+	description: "Summary description for {MA_SHARED_MAIN_WINDOW}."
+	author: ""
+	date: "$Date$"
+	revision: "$Revision$"
+
+class
+	MA_SHARED_MAIN_WINDOW
+
+feature -- Access
+
+	main_window: detachable MA_WINDOW
```

Comment:
If we can avoid it, I would prefer. If we have to keep it, let's make it attached by default. Meaning that the MA_WINDOW should be created right away in the once.